### PR TITLE
Add ItchySats App

### PR DESCRIPTION
**Pull request by 1010Tom**
ItchySats enables trustless derivatives on Bitcoin using DLCs.

### CODE REVIEW

File: apps/registry.json [hunk 1]
```diff
@@ -615,5 +615,30 @@
...
+        "website": "https://itchysats.network",
+        "dependencies": [
+            "bitcoind",
+            "electrum"
+        ],
+        "repo": "https://github.com/itchysats/itchysats",
+        "support": "https://github.com/itchysats/itchysats/issues",
+        "port": 7331,
+        "gallery": [
+            "1.jpg",
```
Comment (da-kami) [inline apps/registry.json:637]: I think those should either be provided of removed

File: apps/itchysats/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+version: "3.7"
+
+services:
+  web:
+    # TODO: get latest release from https://github.com/itchysats/itchysats/pkgs/container/itchysats%2Ftaker
+    image: docker pull ghcr.io/itchysats/itchysats/taker:latest@sha256:dba750409d9446ba34ac5682a9bb7f44b9084f506d1971203773e01ba282c361
```
Comment (da-kami) [inline apps/itchysats/docker-compose.yml]:
```suggestion
    image: ghcr.io/itchysats/itchysats/taker:latest@sha256:dba750409d9446ba34ac5682a9bb7f44b9084f506d1971203773e01ba282c361
```

File: apps/itchysats/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    image: ghcr.io/itchysats/itchysats/taker:latest@sha256:dba750409d9446ba34ac5682a9bb7f44b9084f506d1971203773e01ba282c361
+    restart: on-failure
+    stop_grace_period: 1m
+    ports:
+      - ${APP_ITCHYSATS_PORT}:8000
+    volumes:
+      - ${APP_DATA_DIR}/data:/data
+    command:
+      - --maker=mainnet.itchysats.network:9999
+      - --maker-id=69a42aa90da8b065b9532b62bff940a3ba07dbbb11d4482c7db83a7e049a9f1e
```
Comment (1010Tom) [inline apps/itchysats/docker-compose.yml]: The correct id is: 
```suggestion
      - --maker-id=7e35e34801e766a6a29ecb9e22810ea4e3476c2b37bf75882edf94a68b1d9607
```

**Comment by da-kami** [comment]
Install the plugin when being in `umbrel` folder on the raspi:

```
sudo scripts/update/update --repo itchysats/umbrel#itchysats
```

**Comment by 1010Tom** [comment]
How to open PR: https://github.com/getumbrel/umbrel/blob/master/apps/README.md

**Comment by bonomat** [comment]
already merged https://github.com/getumbrel/umbrel/pull/1149

**Pull request closed without merging**